english/readme.txt

```diff
@@ -0,0 +1 @@
+Some story?
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.